### PR TITLE
[doc] update capabilities (#5550)

### PR DESCRIPTION
**Pull request by frapuks**
update capabilites table :

- Dashboard & investigations
image

- Data sharing & ingestions
image

- Administration
image

### CODE REVIEW

File: docs/administration/users.md [hunk 1]
```diff
@@ -14,33 +14,42 @@ Roles are used in the platform to grant the given groups with some **capabilitie
...
+| `Connectors API usage: register, ping, export push ...`        | Connectors specific permissions for register, ping, push export files, etc.             |
+| `Access data sharing`                                          | Access and consume data such as TAXII collections, CSV feeds & live streams.            |
+| &nbsp;&nbsp;`Manage data sharing`                              | Share data such as TAXII collections, CSV feeds & live streams or custom dashboards.    |
+| `Access ingestion`                                             | Access (read only) Remote OCTI streams TAXII feeds, RSS Feeds, CSV Feeds.               |
+| &nbsp;&nbsp;`Manage ingestion`                                 | Create, update, delete any Remote OCTI streams, TAXII feeds, RSS Feeds, CSV Feeds.      |
+| `Manage CSV mappers`                                           | Create, update and delete CSV mappers.                                                  |
+| `Access to admin functionalities`                              | Parent capability allowing you to only view the settings                                |
```
Comment (aHenryJard) [inline docs/administration/users.md:44]:
```suggestion
| `Access to admin functionalities`                              | Parent capability allowing users to only view the settings                                |
```

File: docs/administration/users.md [hunk 2]
```diff
@@ -14,33 +14,42 @@ Roles are used in the platform to grant the given groups with some **capabilitie
...
+| &nbsp;&nbsp;&nbsp;&nbsp;`Manage public dashboards`             | Manage public dashboards                                                                |
+| `Access connectors`                                            | Read information in the `Data > Connectors` section.                                    |
+| &nbsp;&nbsp;`Manage connector state`                           | Reset the connector state to restart ingestion from the beginning.                      |
+| `Connectors API usage: register, ping, export push ...`        | Connectors specific permissions for register, ping, push export files, etc.             |
+| `Access data sharing`                                          | Access and consume data such as TAXII collections, CSV feeds & live streams.            |
+| &nbsp;&nbsp;`Manage data sharing`                              | Share data such as TAXII collections, CSV feeds & live streams or custom dashboards.    |
+| `Access ingestion`                                             | Access (read only) Remote OCTI streams TAXII feeds, RSS Feeds, CSV Feeds.               |
```
Comment (aHenryJard) [inline docs/administration/users.md:41]:
```suggestion
| `Access ingestion`                                             | Access (read only) remote OCTI streams TAXII feeds, RSS feeds, CSV feeds.               |
```
it's either all with capital `F` or all with `f`

File: docs/administration/users.md (hunk 1 above)
Comment (frapuks) [reply docs/administration/users.md]: Done

File: docs/administration/users.md (hunk 2 above)
Comment (frapuks) [reply docs/administration/users.md]: Done

**Comment by frapuks** [comment]
waiting to updating "dashboard & investigations" capability before merged

**Pull request merged**